content-team/working-docs/social-media-policy.md

```diff
@@ -0,0 +1,83 @@
+# Social Media Strategy and Best Practices
+DD&I Content Team
+
+## Channels
+- [Twitter](https://twitter.com/drupaldiversity, "Drupal Diversity Twitter")
+- [Instagram](https://www.instagram.com/drupaldiversity/, "Drupal Diversity Instagram" )
+- [Drupaldiversity.com blog](https://drupaldiversity.github.io/, "Drupal Diversity Blog")
+- Newsletter 
+
+## Purpose
+
+- To cultivate a more inclusive Drupal community.
+- To give recognition to the diverse voices we have.
+- Transparency, public participation and collaboration.
+- To foster transparency, public participation, and collaboration."
+
+## Audience
+
+- For the community.
+- For the volunteers who contribute to our issue queue.
+- For mentors, and teachers in teh Drupal community.
+- For people who organize and attend Drupal events.
+
+## Content
+
+We want to post content that is related to diversity and inclusion (age/race/gender/religion/more) and/or tech and various places where these issues cross over. Sharing content with different perspectives and different voices allows us to learn and grow in our understanding.
+
+We want people in the community to feel included. By being more aware of how we treat each other and being intentional about the kinds of things we post, we can demonstrate that awareness.
+
+I also think we should look through the list of articles we’ve been compiling to read and see what should be shared, and make sure we are sharing factual resources as well - various information on diversity 101, links to sessions, presentations, articles, faqs, etc.  
```

Comment:
Maybe clean this up? i think this was my bullet point. How about:

We want to share the content that we are consuming, as well - links that we have compiled, links that are shared in the group, books that we read or recommend, and articles that we read. We also want to share information on basic Diversity 101, links to related sessions and presentations, as well as other factual resources.